gmail/Module.md

```diff
@@ -91,40 +91,90 @@ messageRequest.messageBody = "Email Message Body Text";
 //Set the content type of the mail as TEXT_PLAIN or TEXT_HTML.
 messageRequest.contentType = gmail:TEXT_PLAIN;
 //Send the message.
-var sendMessageResponse = gmailEP->sendMessage(userId, messageRequest);
+var sendMessageResponse = gmailClient->sendMessage(userId, messageRequest);
 ```
-The response from `sendMessage` is either a string tuple with the message ID and thread ID 
-(if the message was sent successfully) or a `error` (if the message was unsuccessful). The `match` operation can be 
-used to handle the response if an error occurs.
+
+The response from `sendMessage` is either a string tuple with the message ID and thread ID (if the message was sent successfully) or an `error` (if sending the message was unsuccessful).
+
 ```ballerina
 if (sendMessageResponse is (string, string)) {
-    //If successful, returns the message ID and thread ID.
-    string messageId = "":
-    string threadId = "":
+    // If successful, print the message ID and thread ID.
+    string messageId = "";
+    string threadId = "";
     (messageId, threadId) = sendMessageResponse;
```

Comment:
We can now say 
```suggestion
    (string, string) (messageId, threadId) = sendMessageResponse;
```

We don't have to declare a separate variables for destructuring.